inference-engine/tests/functional/plugin/cpu/single_layer_tests/pad.cpp

```diff
@@ -80,11 +112,13 @@ const auto cpuParams_nCdhw8c = CPUSpecificParams {{nCdhw8c}, {nCdhw8c}, {}, {}};
 const auto cpuParams_nhwc = CPUSpecificParams {{nhwc}, {nhwc}, {}, {}};
 const auto cpuParams_ndhwc = CPUSpecificParams {{ndhwc}, {ndhwc}, {}, {}};
 
+const auto cpuParams_nchw = CPUSpecificParams {{nchw}, {nchw}, {}, {}};
```

Comment:
maybe I missed smt, but could you also add tests for blocked layout and dynamic shapes?

Reply:
There were tests only for [nChw8c](https://github.com/openvinotoolkit/openvino/blob/5f480ace624299933b169d9978667c188784f649/inference-engine/tests/functional/plugin/cpu/single_layer_tests/pad.cpp#L275) (because a lot of the tests) . But I have just added the tests with nChw16c